GiniVision/Classes/Core/ValidatedDocument.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ValidatedDocument.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 4/12/18.
+//
+
+import Foundation
+
+public struct ValidatedDocument {
```

Comment:
What's the point here? A Validated document is a wrapper around the Gini VisionDocument that also has information about the validation and the state?

It's not really easily understandable and maybe even confusing.

Reply:
I couldn't come up with a better naming. Do you have any idea?

Reply:
I would argue that it is confusing to have it there.
Did you create it after I commented that a document shouldn't have an error because it's a breach of capsulation?

Reply:
I think that the problem here is just the name, since it is not a `GiniVisionDocument` but a container for the document and associated value (an error in this case)